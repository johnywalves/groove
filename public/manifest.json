{
  "name": "Groove",
  "short_name": "Groove",
  "description": "Platform for drum groove practice",
  "background_color": "#0e4266",
  "theme_color": "#e0138c",
  "display": "fullscreen",
  "author": "Johny W. Alves",
  "start_url": "/",
  "public_url": "https://groove.johnywalves.com.br/",
  "thumbnail": "https://groove.johnywalves.com.br/img/thumbnail.jpg",
  "icons": [
    {
      "src": "https://groove.johnywalves.com.br/img/icon-192.png",
      "type": "image/png",
      "sizes": "192x192"
    },
    {
      "src": "https://groove.johnywalves.com.br/img/icon-512.png",
      "type": "image/png",
      "sizes": "512x512"
    }
  ]
}
