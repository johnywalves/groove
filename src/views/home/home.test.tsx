import '@testing-library/jest-dom'

import { render, screen } from '@testing-library/react'

import manifest from '../../../public/manifest.json'

import Home from '.'

describe('<HomeView />', () => {
  it('should render the heading', () => {
    const { container } = render(<Home />)

    expect(
      screen.getByRole('heading', { name: manifest.name })
    ).toBeInTheDocument()

    expect(container).toMatchSnapshot()
  })
})
