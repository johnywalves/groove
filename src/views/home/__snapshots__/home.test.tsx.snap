// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeView /> should render the heading 1`] = `
.c5 {
  padding: 2rem;
  color: var(--color-neutral-100);
}

.c5 span {
  color: var(--color-danger);
}

.c5 a {
  color: var(--color-primary-pure);
}

.c5 a:hover {
  text-decoration: underline;
}

.c0 {
  width: 100%;
  min-height: 100vh;
  padding: var(--30px);
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  gap: 1rem;
}

.c1 {
  width: min(var(--300px),50%);
}

.c2 {
  color: var(--color-primary-pure);
  font-size: 2rem;
  text-align: center;
}

.c3 {
  font-size: 1.5rem;
  line-height: 140%;
  font-weight: 400;
  width: min(var(--720px),80%);
}

.c4 {
  font-size: 1.5rem;
}

@media (max-width:980px) {
  .c3 {
    width: 100%;
  }
}

<div>
  <section
    class="c0"
  >
    <img
      alt="Image with curly brackets in magenta with the text React Next Boilerplate"
      class="c1"
      src="/img/logo.svg"
    />
    <h1
      class="c2"
    >
      Groove
    </h1>
    <h2
      class="c3"
    >
      Platform for drum groove practice
    </h2>
    <p
      class="c4"
    >
      just run
    </p>
  </section>
  <div
    class="c5"
  >
    <p>
      Feito com 
      <span>
        â™¥
      </span>
       por
       
      <a
        href="https://johnywalves.com.br/"
      >
        Johny W. Alves
      </a>
    </p>
  </div>
</div>
`;
